Plugins/GitHub3/GitHub3Plugin.cs

```diff
@@ -145,6 +148,31 @@ public IReadOnlyList<IHostedRepository> GetMyRepos()
             return GitHub.getRepositories().Select(repo => (IHostedRepository)new GitHubRepo(repo)).ToList();
         }
 
+        public string OwnerLogin => GitHub.getCurrentUser()?.Login;
+
+        public async Task<string> AddUpstreamRemoteAsync(IGitModule module)
```

Comment:
⚠️  We mustn't do it like this.

A git module is passed via `Register` event:
https://github.com/gitextensions/gitextensions/blob/98747b18124c482187b4a0f06b94df985103e12c/Plugins/GitHub3/GitHub3Plugin.cs#L101-L107

Reply:
Done. But that obliged me to do the same thing for the other already existing methods (in a new commit) 😉